src/assembler/parser/index.ts

```diff
@@ -78,6 +80,24 @@ export function parse(code: string): ICodeFile {
       name: 'Comment',
       pattern: COMMENT
     },
+    {
+      name: 'SPACE',
```

Comment:
name as others camelcase. Perhaps spaceOrFillInstruction or so

Reply:
Changed it